src/ui/public/state_management/state.js

```diff
@@ -205,7 +205,11 @@ export function StateProvider(Private, $rootScope, $location, config, kbnUrl) {
    */
   State.prototype._parseQueryParamValue = function (queryParam) {
     if (!isStateHash(queryParam)) {
-      return rison.decode(queryParam);
+      try {
+        return rison.decode(queryParam);
+      } catch(err) {
+        return null;
+      }
```

Comment:
__question__: I didn't add `try/catch` for `JSON.parse` below as I'm not sure if we can have malformed content in the session storage. To me it's something that is hard to do coincidentally. But what do you think?